index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang = "en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Portfolio">
+        <meta name="viewport" content='width=device-width, inital-scale=1, shrink-to-fit=no'>
+        <title>Cadence Chen</title>
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    </head>
+    <body>
+        <section class="introduction-page">
+                <header>
+                    <div id="myLinks">
+                        <a class="headerLink" href="index.html">Home</a>
+                        <a class="headerLink" href="about.html">About</a>
+                        <a class="headerLink" href="portfolio.html">Portfolio</a>
+                        <a class="headerLink" href="hobby.html">Hobby</a>
+                    </div>
+                </header> 
+      
+
+               <div class="introduction">
+                    <p>Hi, my name is Cadence</p>
+                    <h1>I Build What People Need</h1>
+                    <p>I care about what helps people.</p>
+                    <p> I focus on what people need and build applications that empower and impact people's life</p>
+               </div>
+      
+               <div class="characteristics">
+                    <p class=characteristic>Love<span> what you do</span></p>
+                    <p class=characteristic>Care<span> what impact to people life</span></p>
+                    <p class=characteristic>Focus<span> what your motivation</span></p>
```

Comment:
Make sure you have quotation marks around the class attribute. Also, the parent element would normally have a different class name than the child elements. 
```suggestion
                    <p class="characteristic">Love<span> what you do</span></p>
                    <p class="characteristic">Care<span> what impact to people life</span></p>
                    <p class="characteristic">Focus<span> what your motivation</span></p>
```